--- orig/dsdt.dsl	2016-05-07 01:35:41.000000000 -0400
+++ mod/dsdt.dsl	2016-05-07 01:10:46.000000000 -0400
@@ -5,7 +5,7 @@
  * 
  * Disassembling to symbolic ASL+ operators
  *
- * Disassembly of dsdt.dat, Sat May  7 01:35:41 2016
+ * Disassembly of dsdt.dat, Sun May  1 16:38:14 2016
  *
  * Original Table Header:
  *     Signature        "DSDT"
@@ -433,9 +433,10 @@
                 IO (Decode16,
                     0x0070,             // Range Minimum
                     0x0070,             // Range Maximum
-                    0x01,               // Alignment
-                    0x08,               // Length
+                    0x00,               // Alignment
+                    0x02,               // Length
                     )
+				IRQNoFlags(){10}
             })
         }


